src/components/LeaderBoard/Leaderboard.jsx

```diff
@@ -4,7 +4,7 @@ import { isEqual } from 'lodash';
 import { Link } from 'react-router-dom';
 import { Table, Progress, Modal, ModalBody, ModalFooter, ModalHeader, Button } from 'reactstrap';
 import Alert from 'reactstrap/lib/Alert';
-import { hasLeaderboardPermissions, assignStarDotColors, showStar } from 'utils/leaderboardPermissions';
+import { hasLeaderboardPermissions, assignStarDotColors, showStar, showBlueTime } from 'utils/leaderboardPermissions';
```

Comment:
What is showBlueTime? seems redundant.

Reply:
Hey @cvtqx, it is redundant. I forgot to remove that piece of code. I created a function for it and decided not to use it afterwards. I have removed it now.